### DISCUSSION TITLE
Is it possible to sort the CPUs by `Use%`?

### DISCUSSION THREAD

**Discussion by Integralist** [ClementTsang/bottom#625] [General]
Hello 👋🏻

Just wondering if the CPU widget is able to be sorted (descending/ascending) by the `Use%` column?

I have 8 cores and although the graph (when set to `All`) shows all the cores and which ones are being used the most it would be nice at a glance to see that in the list of CPUs next to it.

Thanks.

**Comment by ClementTsang** [comment]
Some of the (very delayed) work I'm doing right now on a refactor would add support for this, so hopefully soon!
